doc/cookbook/source/examples/binary_classifier/averaged_perceptron.rst

```diff
@@ -0,0 +1,61 @@
+===================
+Averaged Perceptron
+===================
+
+The Averaged Perceptron is an online binary classifier which is used to determine whether an input,
+namely a feature vector :math:`\mathbf{x}`, belongs to some specific class or not. It is an extension
+of the simple perceptron algorithm because it uses the `average weight` vector
```

Comment:
Make it simpler: "It is an extension of the standard Perceptron; it uses the averaged weight and bias."